frontend/src/routes.tsx

```diff
@@ -55,6 +56,10 @@ const codePairRoutes: Array<CodePairRoute> = [
 				path: "member",
 				element: <MemberIndex />,
 			},
+			{
+				path: "profile",
+				element: <ProfileIndex />,
+			},
```

Comment:
Sorry, I think that it's better to move `/profile`.

From a service planning perspective, it seems better if Profiles are not affiliated with any Workspace.

Reply:
i have more advice for this problem 

when i click "Member" button in drawers after move '/profile', i move undefind page.
because profilePage doesn't have `/:workspaceSlug`. so they move '/undefind/members"

![image](https://github.com/user-attachments/assets/c385c60a-a080-4527-a0d6-53183b896163)

Reply:
i think we have three choice 

1. use path parameter "/:workspaceSLug" (56ad9ac)[https://github.com/yorkie-team/codepair/pull/237/commits/56ad9ac95c41d284750a00005e4ba34b0a4debb0]

2. use another layout 
- figma design look like workspaceLayout. so i use same layout

3. use 404 Page when path parameter isn't 
- it think 404 page will be good answer [#137 Issue](https://github.com/yorkie-team/codepair/issues/137)

if you have another good idea, please advice to me. 
(last commit will be soft reset. it is just explain code)